lib/application_insights/rack/track_request.rb

```diff
@@ -24,6 +24,11 @@ def initialize(app, instrumentation_key, buffer_size = 500, send_interval = 60)
       # Track requests and send data to Application Insights asynchronously.
       # @param [Hash] env the rack environment.
       def call(env)
+        # Build a request ID, incorporating one from our request if one exists.
+        id = rand(16**32).to_s(16)
```

Comment:
nit: move the id to the request_id_header function, no need the extra parameter

Reply:
I will do this 👍